build.gradle

```diff
@@ -34,17 +32,11 @@ configurations {
 }
 
 dependencies {
-    implementation group: 'com.ibm.cloud', name: 'cloudant', version: '0.4.3'
-    compileOnly group: 'org.apache.kafka', name: 'connect-api', version: "${kafkaVersion}"
-    compileOnly group: 'org.apache.kafka', name: 'connect-json', version: "${kafkaVersion}"
-    compileOnly group: 'org.apache.kafka', name: 'connect-transforms', version: "${kafkaVersion}"
-    compileOnly group: 'org.slf4j', name: 'slf4j-api', version: '1.7.36'
-    testImplementation group: 'junit', name: 'junit', version: '4.12'
-    testImplementation group: 'org.powermock', name: 'powermock-api-easymock', version: '1.6.4'
-    testImplementation group: 'org.easymock', name: 'easymock', version: '3.4'
-    testImplementation group: 'com.carrotsearch', name: 'junit-benchmarks', version: '0.7.2'
+    implementation(libs.cloudant)
+    compileOnly(libs.bundles.kafkaProvided)
+    testImplementation(libs.bundles.testDeps)
     // for logging output when running tests
-    testRuntimeOnly group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.36'
+    testRuntimeOnly(libs.slf4j.simple)
```

Comment:
```suggestion
    testRuntimeOnly(libs.slf4j-simple)
```

Reply:
Apparently the `.` separated form is the [type-safe accessor](https://docs.gradle.org/current/userguide/platforms.html#sub:mapping-aliases-to-accessors) for the libs in the catalog which have alias with `-` separator even though they recommend using `-` separator 🤷

As per those docs, I could replace them all with camel case and avoid the generator of type-safe names entirely if that is preferred.

Reply:
switched to camel case in [e8dd85a](https://github.com/IBM/cloudant-kafka-connector/pull/164/commits/e8dd85a31d180f0e0112dd187b0f3b215107eaa3) to avoid any future confusion.